greedy_algorithms/dijkstra.cpp

```diff
@@ -32,7 +41,14 @@ class Graph {
         this->edges[src][dst] = weight;
     }
 };
-// Utility function to find minimum distance vertex in mdist
+/**
+ * Function to find minimum distance vertex in mdist.
+ * @param mdist[] array of minimum distances to vertices
+ * @param vset[] array of vertices, where vset[i] is a bool that indicates
+ * whether the vertex i is included in the shortest path tree
+ * @param int number of vertices
+ * @return minimum distance vertex
+ */
 int minDistance(int mdist[], bool vset[], int V) {
     int minVal = INT_MAX, minInd = 0;
```

Comment:
use of `INT_MAX` is ok in C and if the integer size is known. However, C++ provides a better alternative:
```suggestion
    int minVal = std::numeric_limits<int>::infinity, minInd = 0;
```
This needs `#include <limits>` to be included at the top